### ISSUE TITLE
[BUG] - Display error on Amazon Linux

### ISSUE THREAD

**Issue by bitcoin-tools** [bitcoin-tools/nodebuilder#817]
### Expected Behavior

```
#11 9.037   Validating the signatures...10 good.
```

### Actual Behavior

```
#11 9.037   Validating the signatures... /bin/sh: line 788: pgrep: command not found
#11 9.782 /bin/sh: line 789: pgrep: command not found
#11 9.782 10 good.
```

### Steps to Reproduce

1. 
2. 
3.

**Comment by bitcoin-tools** [comment]
Would be easier identified with #395

**Comment by bitcoin-tools** [comment]
Fixed with #837.
